components/Ink/src/private/MDCInkLayer.m

```diff
@@ -379,16 +432,40 @@ - (void)exit:(BOOL)animated {
   [self addAnimation:_backgroundOpacityAnim forKey:kInkLayerBackgroundOpacityAnim];
 }
 
+- (void)clearAnimations {
```

Comment:
Could this be called removeAllAnimations? Or is that conflicting with a UIKit API?